osu.Game/Screens/OnlinePlay/Multiplayer/Match/MatchStartControl.cs

```diff
@@ -251,6 +254,28 @@ private void updateState()
             });
         }
 
+        public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
```

Comment:
You probably want a `!e.Repeat` guard here to avoid spamming ready/unready indefinitely after you hold the shortcut down (same for the other hotkey).

Reply:
Oops, I guess I didn't hold the keys long enough for that to happen during my testing. I just added that.